### ISSUE TITLE
Staging Review finding: `aria-label` incorrectly coded for "Check details and resolve this debt" links

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#107825]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Financial Management
**Product:** Debt Portal
**Feature:** One VA Debt Letter

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When viewing [current debts for a user](https://staging.va.gov/manage-va-debt/summary/debt-balances), each card for each debt has a link that reads "Check details and resolve this debt." Each also has an `aria-label` on the `va-link` component that makes the link unique for screen reader users.

Directly adding an ARIA attribute to a web component has spotty support, and may not be announced by some screen readers. For the [va-link component](https://design.va.gov/storybook/?path=/docs/components-va-link--docs), use the `label` prop instead. That will populate the `aria-label` in the shadow DOM and should be more consistently supported.

<details><summary>Link, screenshot or steps to recreate</summary>

Currently coded as:
```
<va-link active="" data-testid="debt-details-button" href="/manage-va-debt/summary/debt-balances/details/71166" text="Check details and resolve this debt" aria-label="Check details and resolve this Post-9/11 GI Bill debt for books and supplies" class="hydrated"></va-link>
```

</details>

## Recommended action

Use the `label` prop on the `va-link` component, which will then populate the `aria-label` on the link in the shadow DOM.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by HeatherRienks** [comment]
Feedback has been addressed!
